app/controllers/albums.js

```diff
@@ -0,0 +1,17 @@
+const { getAlbums, getAlbum } = require('../services/get_albums');
+const { getPhotos } = require('../services/get_photos');
+
+exports.albums = (req, res, next) => {
+  const albumId = req.params.id;
+  const respond = albumId ? getAlbum : getAlbums;
+  return respond(albumId)
+    .then(response => res.json(response.data))
+    .catch(next);
+};
+
+exports.photos = (req, res, next) => {
+  const albumId = req.params.id;
+  return getPhotos(albumId)
+    .then(response => res.json(response.data))
```

Comment:
`.data` should be resolved by service, in that way if you change axios per request-promise, controllers don't have any changes